### ISSUE TITLE
Punctuation: extra or missing periods in some phrases

### ISSUE THREAD

**Issue by terracoda** [phetsims/balloons-and-static-electricity#327]
Tested latest build from phettest on Nov 8, 2017, with Safari and Voice Over and found some punctuation issues.

**Extra or missing periods here and there (low priority)**
- [x] "Near sweater (need period here)", 
- [x] On second instance of "At center of Play Area (need period here)", 
- [x] Sweater rubbing: "More pairs of charges up and to the left (need period here)", 
- [x] Dragging and reaching wall: "At wall. No transfer of charge. Negative charges in wall move away from Yellow Balloon a little bit (need period here) Positive charges do not move. Yellow balloon has a few more negative charges than positive charges (need period here) Wall has many pairs of negative and positive charges.(remove extra period here)",
- [x] Rubbing on sweater: "Sweater has many more positive charges than negative charges.(remove extra period here)"
- [x] Last rub on sweater: "Sweater has no more negative charges, only positive charges.(remove extra period)"

**Comment by jessegreenberg** [comment]
Thanks very much @terracoda, these issues are fixed.
